charts/spark-operator-chart/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: spark-operator
 description: A Helm chart for Spark on Kubernetes operator
-version: 1.2.15
-appVersion: v1beta2-1.4.6-3.5.0
+version: 1.2.16
+appVersion: v1beta2-1.4.6
```

Comment:
I: The Spark version is separate from the Operator version.  New image tag format is consistent with the old one, but with parameterized Spark version rather than hardcoding it here.